### PR TITLE
Added logging of caught exceptions in REST services.

### PR DESCRIPTION
**Pull request by craigpell**

### CODE REVIEW

File: api-errors-spring/build.gradle [hunk 1]
```diff
@@ -18,6 +18,9 @@ apply from: "$rootDir/gradle/bintray.gradle"
 
 dependencies {
     compile project(':api-errors-core')
+
+    compile group: 'org.slf4j',           name: 'slf4j-api',        version: '1.7.25'
+
```
Comment (JoshMcCullough) [inline api-errors-spring/build.gradle]: I don't think we specifically use slf4k in the other projects. 

```
compile     group: 'org.lognet',                            name: 'grpc-spring-boot-starter',       version: '2.1.4'
```

????

Comment (craigpell) [reply api-errors-spring/build.gradle]: rampart, emailr, clarakm-mail, clarakm-mailimporter, clarakm-aws-events, clarakm-geoconstraint, and clarakm-satmap all do it.

Comment (JoshMcCullough) [reply api-errors-spring/build.gradle]: As you were...

**Comment by fitzyjoe** [comment]
Nice

**Pull request merged**